### PR TITLE
Remove dependency on uvloop

### PR DESCRIPTION
**Pull request by llob**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,3 +24,8 @@ documentation generation, and in the search use case, the function signature its
 
 The resultant wrapper functions have signatures that adhere to the FHIR specification, and they
 forward the requests on to the handlers provided by the developer.
+
+# Contributions
+
+If you would like to contribute to fhirstarter, consider discussing your contribution on GitHub,
```
Comment (csande) [inline CONTRIBUTING.md:30]: Just to be consistent with the rest of the file

```suggestion
If you would like to contribute to FHIRStarter, consider discussing your contribution on GitHub, 
```

Comment (llob) [reply CONTRIBUTING.md:30]: In README.md it's consistently FHIRStarter. Should I make it consistent across both files?

Comment (csande) [reply CONTRIBUTING.md:30]: I checked and except for references in code/URLs (lowercase), I think it's already consistent, except for this new change.

File: pyproject.toml [hunk 2]
```diff
@@ -37,7 +37,6 @@ fastapi = "^0.101.0"
 lxml = "^4.9.3"
 pydantic = ">=1.10.12,<2.0.0"
 python-multipart = "^0.0.6"
-uvloop = "^0.17.0"
```
[the remark this answers is not in the document]
Comment (llob) [reply pyproject.toml:40]: I ran `poetry lock --no-update`, but it does not seem to remove uvloop.
I removed the .lock file and generated a fresh one from the new pyproject.toml file. That still includes uvloop.
I am not familiar with poetry, so not sure how to debug.

[the remark this answers is not in the document]
Comment (csande) [reply pyproject.toml:40]: OK, just wanted to make sure. I think it's getting pulled in by other packages.

**Review by csande — APPROVED** [review]

**Pull request merged**